python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -274,6 +274,7 @@ def run(
         min_repeat_ms=1,
         cooldown_interval_ms=0,
         repeats_to_cooldown=1,
+        sort_by_time=True,
```

Comment:
Please, add documentation and then LGTM

Reply:
Done. I didn't save it the first time ;)